updates readme and variables

tcskill opened the pull request:
updated readme, removed maximo namespace from main - and associated vars with it

tcskill committed:
updates readme and variables

tcskill committed:
adds manage instructions

tcskill committed:
removes manage module readme

tcskill committed:
formatting cleanup

tcskill committed:
fixes generated name

tcskill committed:
adds back portworx file var

seansund reviewed (changes requested):
Two things:
1. All of these changes need to be made in the automation-solutions repo under boms/software/maximo. Otherwise they will all get blown away next time I regenerate the repo
2. I've added logic to setup-workspace.sh to check for the existence of the portworx storage class in the cluster first before adding the portworx storage layer to the workspace

tcskill commented:
@seansund thanks sean, I agree on 1. I changed the bom already in the automation repo to not generate the extra namespace and removed the vars not needed in there. so we should be good
